src/uhttp.c

```diff
@@ -202,9 +203,11 @@ static int process_header_line(const unsigned char* buffer, size_t len, size_t*
                     free(headerKey);
                     headerKey = NULL;
                 }
-                headerKey = (char*)malloc(keyLen + 1);
-                if (headerKey == NULL)
+                size_t malloc_size = safe_add_size_t(keyLen, 1);
+                if (malloc_size == SIZE_MAX ||
+                    (headerKey = (char*)malloc(malloc_size)) == NULL)
                 {
+                    LogError("Cannot malloc memory, size:%zu", malloc_size);
```

Comment:
Could the message be a little more specific please, for systems where the files and line numbers are not printed?

```suggestion
                    LogError("Cannot malloc headerKey, size:%zu", malloc_size);
```